### DISCUSSION TITLE
Welcome to Swift Discussions!

### DISCUSSION THREAD

**Discussion by nicped** [dynamicweb/Swift#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ufaslq** [comment]
Hi @nicped,

Thanks for the invite, I look forward to participating in this group :)

I have been through the repository very briefly but couldn't find any database file(s) .bak or .bacpac. is it intentionally? or its too early to ask for it.

**Reply by nicped** [reply]
Hi Umar

The bacpac was linked in readme - and after the prerelease yesterday it is now updated.

BR Nicolai
